tests/ccl_test_power.py

```diff
@@ -87,7 +87,10 @@ def loop_over_params(transfer_fn, matter_power, lin, raise_errs):
     print(">>> (%s; %s)" % (transfer_fn, matter_power))
 
     # Loop over parameters
-    for i in [0,2]: #w0_vals.size):
+    for i in [0,2]:
+        if matter_power=='halofit' and (w0_vals[i] != -1. or wa_vals[i] != 0.):
+            continue # Halofit disabled for w != -1
+            
```

Comment:
This seems like a pretty serious handicap.

Reply:
So the previous bersion of CLASS does not have this limitation?